### PR TITLE
Add lastdb datatype for last tool (PR #2749)

### PR DESCRIPTION
**Pull request by rdallet**
In accordance with the review from the PR https://github.com/galaxyproject/tools-iuc/pull/2749

I hope it's ok.

### CODE REVIEW

File: lib/galaxy/datatypes/last.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from galaxy.datatypes.data import Data
+
```
Comment (bgruening) [inline lib/galaxy/datatypes/last.py]: @RomainDallet should be move this into the blast.py file? The tools are somehow connected ...

File: lib/galaxy/datatypes/last.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
```
Comment (bgruening) [inline lib/galaxy/datatypes/last.py]: Can you please add at set_peek and display_peek function as well.

**Comment by RomainDallet** [comment]
Hi, is it ok for you ? Can we merge ?

**Comment by RomainDallet** [comment]
Hi @martenson @bgruening,

Is it good for corrections? Can it be merge?

File: lib/galaxy/datatypes/blast.py [hunk 3]
```diff
@@ -307,3 +307,36 @@ def __init__(self, **kwd):
...
+        """Create HTML content, used for displaying peek."""
+        try:
+            return dataset.peek
+        except Exception:
+            return "LAST database (multiple files)"
+
+    def __init__(self, **kwd):
+        Data.__init__(self, **kwd)
+        self.add_composite_file('lastdb.bck', is_binary=True)
+        self.add_composite_file('lastdb.des', is_binary=False)  # description file
```
Comment (hexylena) [inline lib/galaxy/datatypes/blast.py]:
```suggestion
        self.add_composite_file('lastdb.des', description="Description File", is_binary=False)
```

File: lib/galaxy/datatypes/blast.py [hunk 4]
```diff
@@ -307,3 +307,36 @@ def __init__(self, **kwd):
...
+        self.add_composite_file('lastdb.prj', is_binary=False)  # project resume
```
Comment (hexylena) [inline lib/galaxy/datatypes/blast.py]:
```suggestion
        self.add_composite_file('lastdb.prj', description="Project resume file", is_binary=False)
```

**Review by hexylena — COMMENTED** [review]
Two small suggestions, otherwise looks good! Let's get this into 20.09 :)

File: lib/galaxy/datatypes/blast.py (hunk 4 above)
Comment (rdallet) [reply lib/galaxy/datatypes/blast.py]: OK done! Thanks @hexylena.

**Comment by bgruening** [comment]
Thanks a lot @RomainDallet!

**Pull request merged**